dprint.json

```diff
@@ -0,0 +1,16 @@
+{
+  "excludes": [
+    // These files are auto-generated and don't need to be formatted
+    "./types/vscode"
+  ],
+  "plugins": [
+    "https://plugins.dprint.dev/typescript-0.85.1.wasm"
+  ],
+  "typescript": {
+    "arrowFunction.useParentheses": "preferNone",
+    "indentWidth": 4,
+    "lineWidth": 120,
```

Comment:
Not sure if `dprint` treats `lineWidth` as a hard cap or as a recommendation, but if it works similar to Prettier, it might be worth considering 80 characters instead:

https://prettier.io/docs/en/options.html#print-width

> **For readability we recommend against using more than 80 characters:**
>
> In code styleguides, maximum line length rules are often set to 100 or 120. However, when humans write code, they don’t strive to reach the maximum number of columns on every line. Developers often use whitespace to break up long lines for readability. In practice, the average line length often ends up well below the maximum.
>
> Prettier’s printWidth option does not work the same way. It is not the hard upper allowed line length limit. It is a way to say to Prettier roughly how long you’d like lines to be. Prettier will make both shorter and longer lines, but generally strive to meet the specified printWidth.
>
> Remember, computers are dumb. You need to explicitly tell them what to do, while humans can make their own (implicit) judgements, for example on when to break a line.
>
> In other words, don’t try to use printWidth as if it was ESLint’s [max-len](https://eslint.org/docs/rules/max-len) – they’re not the same. max-len just says what the maximum allowed line length is, but not what the generally preferred length is – which is what printWidth specifies.

Reply:
It's similar to prettier:

> The width of a line the printer will try to stay under. Note that the printer may exceed this width in certain cases.

But, I'm personally of the opinion that 120 is the right value to use in this repo and given that's what we've had here (so many projects had been formatted that way), I don't think it's a good idea to change, in addition to my personal preference.